esrally/resources/metrics-template.json

```diff
@@ -42,6 +42,18 @@
             }
           }
         },
+        "race-id": {
```

Comment:
Just to confirm, are we keeping trial-id  and trial-timestamp sections above for backwards compatibility?

Reply:
Correct. This will be removed with https://github.com/elastic/rally/issues/777.